src/main/resources/schemas/schema-form.json

```diff
@@ -129,6 +129,12 @@
             "title": "Field value",
             "description": "The input value to validate (supports EL)"
           },
+          "isRequired" : {
+            "type" : "boolean",
+            "title": "Required",
```

Comment:
`Required` => `Apply if value present`

Reply:
In both cases pattern apply, if value present. So this title won't give informathion about how it works. Maybe, will I rename title  "Require that value present" or something like that?

Reply:
Just leave it that way, that's ok.